pkg/yang/entry.go

```diff
@@ -1248,6 +1252,37 @@ func (e *Entry) ApplyDeviate(deviateOpts ...DeviateOpt) []error {
 						deviatedNode.ListAttr.MaxElements = math.MaxUint64
 					}
 
+					if musts, exists := devSpec.Extra["must"]; exists {
+						// range through the deviations to apply them
+						for _, mustDeleteInterf := range musts {
+
+							mustDelete := mustDeleteInterf.(*Must)
+
+							found := false
+							var idx int
+							var existingMustInterf interface{}
+							var existingMust *Must
+							for idx, existingMustInterf = range deviatedNode.Extra["must"] {
+								if existingMustInterf.(*Must).Name == mustDelete.Name {
+									existingMust = existingMustInterf.(*Must)
+									if (existingMust.ErrorMessage == nil && mustDelete.ErrorMessage == nil) ||
+										(existingMust.ErrorMessage != nil && mustDelete.ErrorMessage != nil &&
+											existingMust.ErrorMessage.Name == mustDelete.ErrorMessage.Name) {
+										found = true
+										break
+									}
+								}
+							}
+							if found {
```

Comment:
Suggest to switch this around to keep the error case the indented one per go-style -- i.e., `if !found { appendErr(); continue }`.

Reply:
done